etc/tempesta_fw.conf

```diff
@@ -699,4 +699,21 @@
 # Default:
 #   All limits are disabled (the values are set to zero/false/empty).
 
+#
+# TAG: whilelist_mark
```

Comment:
whi**T**elist_mark. Same misprint in lines below. I was surprised to get `ERROR: don't know how to handle: 'whilelist_mark` after copying name from configuration example(:

Reply:
Yeah - copy-paste=(
Thanks! Corrected.